### ISSUE TITLE
NullPointerException in CalcCdrAAProfile routine

### ISSUE THREAD

**Issue by mattflor** [mikessh/vdjtools#52]
A java NullPointerException occurs if I call the CalcCdrAAProfile routine. As far as I can see this seems to happen even before command line arguments are read. Here's what the ` _vdjtools_error.log` says:

```
[Wed Jul 27 11:37:35 CEST 2016 BEGIN]
[Script]
CalcCdrAAProfile
[CommandLine]
executing vdjtools-1.1.0.jar CalcCdrAAProfile -h
[Message]
java.lang.NullPointerException: Cannot invoke method join() on null object
[StackTrace-Short]
com.antigenomics.vdjtools.profile.CalcCdrAAProfile.run(CalcCdrAAProfile.groovy:41)
com.antigenomics.vdjtools.profile.CalcCdrAAProfile$run.call(Unknown Source)
com.antigenomics.vdjtools.misc.ExecUtil.run(ExecUtil.groovy:94)
com.antigenomics.vdjtools.misc.ExecUtil$run.call(Unknown Source)
com.antigenomics.vdjtools.VdjTools.run(VdjTools.groovy:207)
com.antigenomics.vdjtools.VdjTools.main(VdjTools.groovy)
[StackTrace-Full]
java.lang.NullPointerException: Cannot invoke method join() on null object
	at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:88)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:45)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:32)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at com.antigenomics.vdjtools.profile.CalcCdrAAProfile.run(CalcCdrAAProfile.groovy:41)
	at com.antigenomics.vdjtools.profile.CalcCdrAAProfile$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at com.antigenomics.vdjtools.misc.ExecUtil.run(ExecUtil.groovy:94)
	at com.antigenomics.vdjtools.misc.ExecUtil$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:120)
	at com.antigenomics.vdjtools.VdjTools.run(VdjTools.groovy:207)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:233)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1085)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:909)
	at org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:901)
	at org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:884)
	at org.codehaus.groovy.runtime.InvokerHelper.runScript(InvokerHelper.java:406)
	at org.codehaus.groovy.runtime.InvokerHelper$runScript.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:120)
	at com.antigenomics.vdjtools.VdjTools.main(VdjTools.groovy)
[END]

```

**Comment by mikessh** [comment]
Hello! Indeed, this one was fixed in 1.1.1 patch (https://github.com/mikessh/vdjtools/releases/tag/1.1.1) and marked as a known bug in https://github.com/mikessh/vdjtools/releases/tag/1.1.0

**Comment by mattflor** [comment]
Ah, I had missed that info. Thanks for quick response!
